api/pkg/transformer/feast/bigtablestore/client.go

```diff
@@ -0,0 +1,61 @@
+package bigtablestore
+
+import (
+	"context"
+
+	"cloud.google.com/go/bigtable"
+
+	feast "github.com/feast-dev/feast/sdk/go"
+	"github.com/gojek/merlin/pkg/transformer/spec"
+)
+
+// BigTableClient provides the means of retrieving Feast features directly from
+// BigTable without going through Feast Serving.
+type BigTableClient struct {
+	encoder *Encoder
+	tables  map[string]*bigtable.Table
+}
+
+// NewClient instantiate new instance of BigTableClient
+func NewClient(client *bigtable.Client, project string, featureSpecs []*spec.FeatureTable, metadata []*spec.FeatureTableMetadata) *BigTableClient {
```

Comment:
I noticed that `NewClient` is not being wired up yet to `NewDirectStorageClient` , is it going to be a separate PR?